index.bs

```diff
@@ -139,37 +136,40 @@ extensible.
 <button id="pipButton"></button>
 
 <script>
-  let pipWindow;
-  const video = document.getElementById('video');
-  const pipButton = document.getElementById('pipButton');
+  const video = document.getElementById("video");
+  const pipButton = document.getElementById("pipButton");
 
-  pipButton.addEventListener('click', function() {
-    video.requestPictureInPicture()
-    .catch(error => {
+  pipButton.addEventListener("click", async () => {
```

Comment:
I'm not sure it is more readable this way.

Reply:
Doing the `.catch()` (or `.then()`) chaining is not very idiomatic given we have async/await. I'm imagining people copy/pasting this code then needing to adapt it.

Reply:
Noted. Let's keep it this way.